kubernetes/helm-charts/buildfarm/templates/server/deployment.yaml

```
@@ -43,13 +43,13 @@ spec:
             - containerPort: 9090
               name: "metrics"
           livenessProbe:
-            httpGet:
-              path: /
-              port: metrics
+            grpc:
+              port: server-comm
```

As far as I know, this has to be a numbered port.
```suggestion
              port: 8980
```

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe

>Unlike HTTP or TCP probes, you cannot specify the health check port by name, and you cannot configure a custom hostname.

---

Thanks. Good catch!